src/theory/bv/bv_solver_bitblast.cpp

```diff
@@ -408,6 +411,47 @@ void BVSolverBitblast::handleEagerAtom(TNode fact, bool assertFact)
   registeredAtoms.clear();
 }
 
+std::vector<Node> BVSolverBitblast::getProofNodes(proof::DratProof dratProof)
+{
+  NodeManager* nm = NodeManager::currentNM();
+  Node cl = nm->mkBoundVar("cl", nm->booleanType());
+  Node del = nm->mkBoundVar("del", nm->booleanType());
+  Node lastFalseResolution = nm->mkConst<bool>(false);
+
+  std::vector<Node> args;
+  for (const proof::DratInstruction instruction : dratProof.getInstructions())
+  {
+    if (instruction.d_clause.size() == 0 && instruction.d_clause[0].toInt() == 0)
```

Comment:
Rather than using `toInt`, create a `zeroNode` similar to `lastFalseResolution` above (with the integer constant 0) and compare `d_clause[0]` to it.